packages/redux/src/orders/actions/__tests__/fetchOrderShippingAddressChangeRequests.test.ts

```diff
@@ -0,0 +1,90 @@
+import * as actionTypes from '../../actionTypes';
+import { fetchOrderShippingChangeAddressRequests } from '..';
+import { getOrderShippingAddressChangeRequests } from '@farfetch/blackout-client';
+import { INITIAL_STATE } from '../../reducer';
+import {
+  mockOrderShippingAddressChangeRequestsResponse,
+  orderId,
+} from 'tests/__fixtures__/orders';
+import { mockStore } from '../../../../tests';
+
+jest.mock('@farfetch/blackout-client', () => ({
+  ...jest.requireActual('@farfetch/blackout-client'),
+  getOrderShippingAddressChangeRequests: jest.fn(),
+}));
+
+const ordersMockStore = (state = {}) =>
+  mockStore({ orders: INITIAL_STATE }, state);
+const expectedConfig = undefined;
+let store: ReturnType<typeof ordersMockStore>;
+
+describe('fetchOrderShippingChangeAddressRequests() action creator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    store = ordersMockStore();
+  });
+
+  it('should create the correct actions fetch when the get order address change requests procedure fails', async () => {
+    const expectedError = new Error(
+      'fetch order address change requests error',
+    );
+
+    (getOrderShippingAddressChangeRequests as jest.Mock).mockRejectedValueOnce(
+      expectedError,
+    );
+
+    expect.assertions(4);
```

Comment:
Same as the comment in `addOrderShippingAddressChangeRequest`

Reply:
All the actions were removed, so this file no longer exists.